index.html

```diff
@@ -6751,8 +6810,11 @@ <h3 id="Elemental">Elemental</h3>
                </td>
                <td>Rock Flurry
                </td>
-               <td><a class="condition" data-tooltip-container>Immobilise</a> yourself and fire a total of 9 shards of rock at any number of
-                  targets within 50ft. Each deals Light damage.
+               <td>
+                  <strong>Unleash a rapid barrage of stone.</strong>
+                  <p>
+                     <a class="condition" data-tooltip-container>Immobilise</a> yourself and deal Light damage 9 times
```

Comment:
You don't roll to hit?  Just an unmissable 3x Heavy?

Reply:
Correct, rolling 9 times is a nono. On this tho, idk why it's 9, that's a lot. Might just change to 6 for 2*Heavy.